Dockerfile

```diff
@@ -10,19 +12,19 @@ LABEL maintainer="gilbn"
 RUN \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \
-    php8 \
-    php8-sqlite3 \
-    php8-mysqli \
-    php8-pdo_mysql \
-    php8-pdo_sqlite \
-    php8-gd \
-    php8-json \
-    php8-fileinfo \
-    php8-zip \
-    php8-ftp \
-    php8-ldap \
-    php8-tokenizer \
-    php8-intl && \
+    php81 \
```

Comment:
Base already has php81

Reply:
Resolved in https://github.com/linuxserver/docker-xbackbone/pull/21/commits/ea1c221b59f954a2c4177ee96d2f48ce3e8bc430